v3/src/components/axis/models/axis-model.ts

```diff
@@ -162,27 +162,46 @@ export function isNumericAxisModel(axisModel?: IAxisModel): axisModel is INumeri
   return axisModel?.type === "numeric"
 }
 
+export const PercentAxisModel = BaseNumericAxisModel
+  .named("PercentAxisModel")
+  .props({
+    type: types.optional(types.literal("percent"), "percent")
+  })
+  .views(self => ({
+    get lockZero() {
+      return true
+    },
+  }))
+export interface IPercentAxisModel extends Instance<typeof PercentAxisModel> {}
+export interface IPercentAxisModelSnapshot extends SnapshotIn<typeof PercentAxisModel> {}
+export function isPercentAxisModel(axisModel?: IAxisModel): axisModel is IPercentAxisModel {
+  return axisModel?.type === "percent"
+}
+
+// ToDo: It _should_ be possible to have CountAxisModel inherit from PercentAxisModel and get rid of lockZero
+//  but it causes a strange typescript error in graph-content-model.ts
 export const CountAxisModel = BaseNumericAxisModel
   .named("CountAxisModel")
   .props({
     type: types.optional(types.literal("count"), "count")
   })
   .views(self => ({
-    get integersOnly() {
-      return true
-    },
     get lockZero() {
       return true
     },
+    get integersOnly() {
+      return true
+    }
   }))
 export interface ICountAxisModel extends Instance<typeof CountAxisModel> {}
 export interface ICountAxisModelSnapshot extends SnapshotIn<typeof CountAxisModel> {}
 export function isCountAxisModel(axisModel?: IAxisModel): axisModel is ICountAxisModel {
   return axisModel?.type === "count"
 }
 
-export function isNumericOrCountAxisModel(axisModel?: IAxisModel): axisModel is INumericAxisModel | IDateAxisModel {
-  return isNumericAxisModel(axisModel) || isCountAxisModel(axisModel)
+export function isNumericOrCountOrPercentAxisModel(axisModel?: IAxisModel):
+    axisModel is INumericAxisModel | IDateAxisModel | ICountAxisModel | IPercentAxisModel {
```

Comment:
It seems that the goal of this function is to exclude date axes, in which case it should be:
```suggestion
    axisModel is INumericAxisModel | ICountAxisModel | IPercentAxisModel {
```
On the other hand, if we don't care about excluding date axes then it's redundant with `isBaseNumericAxisModel()` below.